hoomd/md/EvaluatorPairALJ.h

```diff
@@ -283,7 +283,14 @@ template<unsigned int ndim> class EvaluatorPairALJ
             // Simulating an ellipsoid just return
             if (N_vertices == 0 && N_faces == 0)
```

Comment:
Minor nit: If you moved this block up a bit you could avoid assigning verts/faces/face_offsets twice. It doesn't really matter since the initial ManagedArrays will be empty anyway, just a note.